templates/articles/_comment_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+<turbo-frame id="comment-{{ object.pk }}">
+    <div class="card">
+        <div class="card-block">
+            <p class="card-text">
+                Are you sure you want to delete your comment "{{ object.body }}"?
+            </p>
+        </div>
+            <div class="card-footer">
+            <form method="post" action="{% url 'delete_comment' object.pk %}">
+                {% csrf_token %}
+                <input type="submit" class="btn btn-sm btn-danger" value="Confirm Delete">
+                <a href="{{ view.get_success_url }}" class="btn btn-default">Cancel</a>
```

Comment:
I'm not sure if this is working, when I click the cancel button nothing happens on the interface.

In the network console, we have this call after the click:
![Screen Shot 2021-01-08 at 10 45 50](https://user-images.githubusercontent.com/1490875/104022165-af243380-519e-11eb-8b98-382f91d425bf.png)

Reply:
Fixed 👍🏻